packages/wrangler/src/index.tsx

```diff
@@ -326,8 +329,44 @@ export async function main(argv: string[]): Promise<void> {
             console.log(
               `✨ Installed @cloudflare/workers-types.\nPlease add "@cloudflare/workers-types" to compilerOptions.types in your tsconfig.json`
             );
-          } else {
-            return;
+          }
+        }
+      }
+
+      if (isTypescriptProject) {
+        if (!fs.existsSync(path.join(process.cwd(), "src/index.ts"))) {
+          const shouldCreateSource = await confirm(
+            `Would you like to create a Worker at src/index.ts?`
+          );
+          if (shouldCreateSource) {
+            const targetDir = path.join(process.cwd(), "src");
```

Comment:
NIT: we don't actually need to use `process.cwd()` everywhere since these `fs` functions will run relative paths against the current working directory by default.

Reply:
aha, noted. it's all over this file, so I'll do them all in one PR right after this.

Reply:
https://github.com/cloudflare/wrangler2/pull/383